client/components/app.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
@LittleJennie 
@cminho921 

Again, a bit draconian as we know that this is going to be stateful eventually, but until it is, components should be functional stateless if the constructor and render methods are empty!

Also, remember the new line at the end of the file!

Reply:
Thanks for pointing this out Ram! I didn't set up eslint until my 2nd or 3rd PR. So the this PR is not following Airbnb style guide at all. I was able to fix the styles in the later PR.